functions/java/modules/module1/pom.xml

```diff
@@ -14,6 +14,18 @@
         <httpclient.version>4.5.14</httpclient.version>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>software.amazon.awssdk</groupId>
+                <artifactId>bom</artifactId>
+                <version>2.21.21</version>
+                <type>pom</type>
+                <scope>import</scope>
```

Comment:
Hey @deki , if we're doing a BOM import, wouldn't we expect to have some dependencies in our POM that no longer specify a version, taking them instead from the BOM? 

I've not used this pattern before so I might be missing something obvious, just going from [the apache documentation](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms)

Reply:
Yes see below `software.amazon.awssdk:s3` and `software.amazon.awssdk:dynamodb` but the main fix here is to make sure we use the same version also for the transitive dependencies (currently it's different... see output of `mvn dependency:list`).

Reply:
You're right - missed those two. It's late !